molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -16,7 +16,7 @@
     <#if googleSignIn>
         <meta name="google-signin-client_id" content="${app_settings.googleAppClientId?html}">
     </#if>
-    <link rel="icon" href="<@resource_href "/img/molgenis.ico"/>" type="image/x-icon">
```

Comment:
Then we should edit the jenkins job.
the reason why so many browsers throw the favicon: not found error is because it should be called favicon.ico and not molgenis.ico